installer/README.md

```diff
@@ -28,76 +115,111 @@ Install script can be executed with three different switches to:
+
+  Usage install [--help|--precheck|--skip_precheck]
+  Usage:
+      install [-h|--help]
+      install [-p|--precheck]
+      install [-s|--skip_precheck]
+  Flags:
+      -h, --help Print the usage of install
+      -p, --precheck Only perform system software requirements for installation
+      -s, --skip_precheck If precheck is not required and directly install
+  ```
+
+## For Pre-check, run below command
+  ```sh
+  installer/install -p
+
+  # Example
+
+  root@root1:~/delfin-demo/delfin$ installer/install -p
+                              OR
+  root@root1:~/delfin-demo/delfin/installer$ ./install --precheck
+  ```
 
 ## Install without pre-check
-```
-<path to installer/install script>/install -s
-Ex:
-$ pwd
-/root/gopath/src/github.com/sodafoundation/delfin
-$ installer/install -s
-
-OR
+```sh
+installer/install -s
 
-$ pwd
-/root/gopath/src/github.com/sodafoundation/delfin/installer
-$ ./install -s
+# Example
 
+root@root1:~/delfin-demo/delfin$ installer/install -s
 ```
 
 ## Execute both pre-check as well as install
-```
-<path to installer/install script>/install
-Ex:
-$ pwd
-/root/gopath/src/github.com/sodafoundation/delfin
-$ installer/install
-
-OR
-
-$ pwd
-/root/gopath/src/github.com/sodafoundation/delfin/installer
-$ ./install
+```sh
+installer/install
 
+# Example
+root@root1:~/delfin-demo/delfin$ installer/install
 ```
 
 # Uninstall
 Running the uninstall script will stop all delfin processes and do cleanup
-```
-<path to installer/install script>/uninstall.sh
-Ex:
-$ pwd
-/root/gopath/src/github.com/sodafoundation/delfin
-$ installer/uninstall
-
+    
+    installer/uninstall
+    
+    # Example
+    root@root1:~/delfin-demo/delfin$ installer/uninstall
+
+
+# Test the running delfin setup
+1. Make sure all delfin process are up and running
+
+    ```
+    ps -ef|grep delfin
+
+    # Example
+    root@root1:~/delfin-demo/delfin# ps -ef |grep delfin
+    root       25856    3570  0 00:21 pts/0    00:00:04 python3 /root/delfin-demo/delfin/installer/../delfin/cmd/api.py --config-file /etc/delfin/delfin.conf
+    root       25858    3570  0 00:21 pts/0    00:00:09 python3 /root/delfin-demo/delfin/installer/../delfin/cmd/task.py --config-file /etc/delfin/delfin.conf
+    root       25860    3570  0 00:21 pts/0    00:00:06 python3 /root/delfin-demo/delfin/installer/../delfin/cmd/alert.py --config-file /etc/delfin/delfin.conf
+    root       25862    3570  0 00:21 pts/0    00:00:00 python3 /root/delfin-demo/delfin/installer/../delfin/exporter/exporter_server.py --config-file /etc/delfin/delfin.conf
+
+    ```
+
+2. Register storages
+
+    POST http://localhost:8190/v1/storages
+
+    body :
+  
+    ```
+      {
+        "vendor": "fake_storage",
+        "model": "fake_driver",
+        "rest": {
+        "host": "127.0.0.1",
+        "port": 8088,
+        "username": "admin",
+        "password": "pass"
+        },
+        "extra_attributes": {
+            "array_id": "12345"
+        }
+       }
+
+    ```
```

Comment:
Add on step here to to pick the storag_id , which is rquired  for performance and alert source registration. You can add alert source regitration also as step 6